hyperSpec/DESCRIPTION

```diff
@@ -4,14 +4,14 @@ Type: Package
 Title: Work with Hyperspectral Data, i.e. Spectra + Meta Information (Spatial,
     Time, Concentration, ...)
 Version: 0.99-20200319
-Date: 2020-03-19
+Date: 2020-05-19
 Maintainer: Claudia Beleites <Claudia.Beleites@chemometrix.gmbh>
 Authors@R: c(
     person("Claudia",   "Beleites",  role = c("aut","cre", "dtc"), email = "Claudia.Beleites@chemometrix.gmbh"),
     person("Valter",    "Sergo",     role = c("aut")),
     person("Alois",     "Bonifacio", role = c ("ctb", "dtc")),
     person("Marcel",    "Dahms",     role = "ctb"),
-    person("Björn",     "Egert",     role = "ctb"),
+    person("Bj<U+00F6>rn",     "Egert",     role = "ctb"),
```

Comment:
Is an encoding issue introduced here?

Reply:
I have tracked this down to R version 4.0.0 RC (2020-04-21 r78267) which I am using.  It only has locale "C" for some reason.  It is corrected in 4.1 but other things have changed there (changes in apply arguments) which affect `hyperSpec` so once #126 is fixed we can move to 4.1 and this locale issue should go away.